src/api/route-services/projects.js

```diff
@@ -42,6 +43,19 @@ class ProjectsService {
     return response;
   }
 
+  async createProject(projectUuid, project) {
+    logger.log(`Creating project with id ${projectUuid}`);
+
+    const experimentId = project.experiments[0];
+
+    await Promise.all([
+      this.createEmptySamplesEntry(projectUuid, experimentId),
```

Comment:
Why do we need to create an empty samples entry? If we upload files, can't the API just save it if there is no samples entry before?

Reply:
If we are doing an update where we just overwite the existing then it is fine.

In our case we are updating an existing element, and creating it empty if it doesn't exist can't be done in the same call that does an update to a nested property (it is actually a problem I already ran into with the processing config back when we didn't create it originally)